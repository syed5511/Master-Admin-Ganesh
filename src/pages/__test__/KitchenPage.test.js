import KitchenPage from "../kitchen";
import getSnapShot from "../../utils/getSnapShot";

test("Kitchen page snapshot", () => {
  getSnapShot(KitchenPage)();
});
