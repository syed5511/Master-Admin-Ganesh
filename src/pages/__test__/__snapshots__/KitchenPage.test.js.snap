// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kitchen page snapshot 1`] = `
Array [
  <nav
    aria-label="breadcrumb"
    className="sc-AxiKw domzKi"
  >
    <ol
      className="breadcrumb"
    >
      <li
        aria-current="page"
        className="breadcrumb-item sc-AxhCb fxcQsP active"
      >
        Registered kitchens
      </li>
    </ol>
  </nav>,
  <div
    className="sc-fznWOq jOfCeA"
  >
    <div
      className="sc-fzoiQi jkvegY"
    >
      <form
        className="sc-fzoyTs crHzLr"
        noValidate={true}
        onSubmit={[Function]}
      >
        <div
          className="sc-fzqNqU fbPvyo row"
        >
          <div
            className="sc-fzoLsD iFBWyk col-md-2 col-sm-4 col-6"
          >
            <div
              className="sc-fzplWN QOzlh"
            >
              <div
                className="sc-fzokOt dPfWvZ"
              >
                <div
                  aria-controls="picky__button"
                  aria-owns=""
                  className="picky "
                  id="picky"
                  role="combobox"
                  tabIndex={0}
                >
                  <button
                    className="picky__input  "
                    data-testid="picky-input"
                    disabled={false}
                    id="picky__button__button"
                    onClick={[Function]}
                    type="button"
                  >
                    <span
                      className="picky__placeholder"
                      data-testid="picky_placeholder"
                    >
                      State
                    </span>
                  </button>
                  <div
                    aria-hidden={true}
                    className="picky__dropdown"
                    hidden={true}
                    id="picky-list"
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="sc-fzoLsD iFBWyk col-md-2 col-sm-4 col-6"
          >
            <div
              className="sc-fzplWN QOzlh"
            >
              <div
                className="sc-fzokOt dPfWvZ"
              >
                <div
                  aria-controls="picky__button"
                  aria-owns=""
                  className="picky "
                  id="picky"
                  role="combobox"
                  tabIndex={0}
                >
                  <button
                    className="picky__input  "
                    data-testid="picky-input"
                    disabled={false}
                    id="picky__button__button"
                    onClick={[Function]}
                    type="button"
                  >
                    <span
                      className="picky__placeholder"
                      data-testid="picky_placeholder"
                    >
                      Kitchen
                    </span>
                  </button>
                  <div
                    aria-hidden={true}
                    className="picky__dropdown"
                    hidden={true}
                    id="picky-list"
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="sc-fzoLsD iFBWyk col-md-2 col-sm-4 col-6"
          >
            <div
              className="sc-fzplWN QOzlh"
            >
              <div
                className="sc-fzokOt dPfWvZ"
              >
                <div
                  aria-controls="picky__button"
                  aria-owns=""
                  className="picky "
                  id="picky"
                  role="combobox"
                  tabIndex={0}
                >
                  <button
                    className="picky__input  "
                    data-testid="picky-input"
                    disabled={false}
                    id="picky__button__button"
                    onClick={[Function]}
                    type="button"
                  >
                    <span
                      className="picky__placeholder"
                      data-testid="picky_placeholder"
                    >
                      Zone no
                    </span>
                  </button>
                  <div
                    aria-hidden={true}
                    className="picky__dropdown"
                    hidden={true}
                    id="picky-list"
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="sc-fzoLsD iFBWyk col-md-2 col-sm-4 col-6"
          >
            <div
              className="sc-fzplWN QOzlh"
            >
              <div
                className="sc-fznZeY jsiuof"
              >
                <input
                  className="form-control"
                  disabled={false}
                  name="item_name"
                  onChange={[Function]}
                  placeholder="Item Name"
                  type="text"
                />
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>,
  <div
    className="table-responsive"
  >
    <table
      className="sc-AxjAm bLuHfC table table-sm table-hover"
    >
      <thead
        className="sc-AxirZ faxrkr"
      >
        <tr>
          <th>
            Kitchen Name
          </th>
          <th>
            State
          </th>
          <th>
            City
          </th>
          <th>
            Zip Code
          </th>
          <th>
            Zone No
          </th>
          <th>
            Status
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            <a
              className="sc-fzpmMD hllTgD"
              onClick={[Function]}
            >
              Syed Kitchen
            </a>
          </td>
          <td>
            Illinios
          </td>
          <td>
            Chicago
          </td>
          <td>
            10143
          </td>
          <td>
            101
          </td>
          <td>
            <div
              className="sc-fznMAR ebGfYL Active"
            >
              Active
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>,
  <ul
    className="sc-AxhUy iwjpuy pagination"
  >
    <li
      className="sc-AxheI fniENP page-item disabled"
    >
      <span
        className="page-link"
        disabled={true}
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          «
        </span>
        <span
          className="sr-only"
        >
          First
        </span>
      </span>
    </li>
    <li
      className="sc-Axmtr hBVTto page-item disabled"
    >
      <span
        className="page-link"
        disabled={true}
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          ‹
        </span>
        <span
          className="sr-only"
        >
          Previous
        </span>
      </span>
    </li>
    <li
      className="sc-AxgMl kDEWZS page-item active"
    >
      <span
        className="page-link"
        disabled={false}
      >
        1
        <span
          className="sr-only"
        >
          (current)
        </span>
      </span>
    </li>
    <li
      className="sc-AxgMl kDEWZS page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        2
      </a>
    </li>
    <li
      className="sc-AxmLO cAKCgh page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
        >
          ›
        </span>
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </li>
    <li
      className="sc-fzozJi jxgYyc page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
        >
          »
        </span>
        <span
          className="sr-only"
        >
          Last
        </span>
      </a>
    </li>
  </ul>,
]
`;
